doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -297,6 +297,7 @@ In order to properly set up the data replication, the slave nodes must be able t
    After removing these groups, the super user created during the security server installation is a member of only one UI privilege group: `xroad-securityserver-observer`. This group allows read-only access to the admin user interface and provides a safe way to use the UI for checking the configuration status of the slave security server. Since admin UI users are UNIX users that are members of specific privilege groups, more users can be added to the read-only group as necessary. Security server installation scripts detect the node type of existing installations and modify user group creation accordingly so as to not overwrite this configuration step during security server updates.
 
    For more information on user groups and their effect on admin user interface privileges in the security server, see the  Security Server User Guide \[[UG-SS](#12-references)\].
+10. It is highly recommended to use autologin-package with Slaves, because inserting PIN-code is needed for Slave to work at all. Installing and using autologin is guided here: [autologin documentation](../Utils/ug-autologin_x-road_v6_autologin_user_guide.md)
```

Comment:
Using autologin feature in production environments might result in security issues. I do not think we need to promote this feature as a highly recommended one.

Reply:
Reply related to https://github.com/ria-ee/X-Road/pull/57#issuecomment-326227279 "_Certificates metric returns information about disabled certificates_":  Original specification of what information to show about certificates in metrics data, did not exclude disabled certificates. It only said that metrics data for all certificates should be fetched. 

However, your proposal for improving the functionality by adding 'active or disable' information included to every certificate listed in the metrics data, is very good. I will create a new issue (to Joint Development backlog) for this one so that the upcoming X-Road release (probably 6.17.x)   will include the enhanced functionality.

Reply:
https://github.com/vrk-kpa/xroad-joint-development/issues/176